### PR TITLE
Add sample for Cloud SQL for SQL Server

### PR DESCRIPTION
**Pull request by shubha-rajan**
Fixes #issue

> It's a good idea to open an issue first for discussion.

- [x] I have followed [Sample Format Guide](https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md)
- [x] `pom.xml` parent set to latest `shared-configuration`
- [x] Appropriate changes to README are included in PR
- [ ] API's need to be enabled to test (tell us)
- [ ] Environment Variables need to be set (ask us to set them)
- [ ] **Tests** pass:   `mvn clean verify` **required**
- [x] **Lint**  passes: `mvn -P lint checkstyle:check` **required**
- [ ] **Static Analysis**:  `mvn -P lint clean compile pmd:cpd-check spotbugs:check` **advisory only**
- [ ] Please **merge** this PR for me once it is approved.

### CODE REVIEW

**Comment by lesv** [comment]
@kolea2 PING

File: cloud-sql/sqlserver/servlet/README.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+To run this application locally, run the following command inside the project folder:
+
+```bash
+mvn jetty:run
+```
+
+Navigate towards `http://127.0.0.1:8080` to verify your application is running correctly.
+
+## Google App Engine Standard
```
Comment (kolea2) [inline cloud-sql/sqlserver/servlet/README.md:42]: any cleanup instructions we should add?

File: cloud-sql/sqlserver/servlet/pom.xml [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          <scanIntervalSeconds>1</scanIntervalSeconds>
+        </configuration>
+      </plugin>
+      <!-- Only required for AppEngine Deployments -->
+      <plugin>
+        <groupId>com.google.cloud.tools</groupId>
+        <artifactId>appengine-maven-plugin</artifactId>
+        <version>2.2.0</version>
+        <configuration>
+          <projectId>samples-testing-shubhadayini</projectId>
```
Comment (kolea2) [inline cloud-sql/sqlserver/servlet/pom.xml:85]: this should probably be a placeholder value?

File: cloud-sql/sqlserver/servlet/src/main/java/com/example/cloudsql/IndexServlet.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      // Most databases cache previously compiled queries, which improves efficiency.
+      PreparedStatement voteStmt =  conn.prepareStatement(
+          "SELECT TOP(5) candidate, time_cast FROM votes ORDER BY time_cast DESC");
+      // Execute the statement
+      ResultSet voteResults = voteStmt.executeQuery();
+      // Convert a ResultSet into Vote objects
+      while (voteResults.next()) {
+        String candidate = voteResults.getString(1);
+        Timestamp timeCast = voteResults.getTimestamp(2);
+        recentVotes.add(new Vote(candidate.trim(), timeCast));
```
Comment (kolea2) [inline cloud-sql/sqlserver/servlet/src/main/java/com/example/cloudsql/IndexServlet.java:63]: feel free to take or leave this suggestion - I would refactor `new Vote(...` out as a variable.

File: cloud-sql/sqlserver/servlet/src/main/java/com/example/cloudsql/IndexServlet.java [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      voteStmt.setString(2, team);
+
+      // Finally, execute the statement. If it fails, an error will be thrown.
+      voteStmt.execute();
+
+    } catch (SQLException ex) {
+      // If something goes wrong, handle the error in this section. This might involve retrying or
+      // adjusting parameters depending on the situation.
+      // [START_EXCLUDE]
+      LOGGER.log(Level.WARNING, "Error while attempting to submit vote.", ex);
```
Comment (kolea2) [inline cloud-sql/sqlserver/servlet/src/main/java/com/example/cloudsql/IndexServlet.java:131]: warning or error here?

File: cloud-sql/sqlserver/servlet/src/main/webapp/WEB-INF/appengine-web.xml [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
+  <threadsafe>true</threadsafe>
+  <runtime>java8</runtime>
+  <env-variables>
+    <env-var name="CLOUD_SQL_INSTANCE_NAME" value="samples-testing-shubhadayini:us-central1:sql-server-test" />
```
Comment (kolea2) [inline cloud-sql/sqlserver/servlet/src/main/webapp/WEB-INF/appengine-web.xml:21]: replace value?

File: cloud-sql/sqlserver/servlet/src/main/webapp/index.jsp [hunk 6]
```diff
@@ -0,0 +1,118 @@
...
+    <div class="nav-wrapper">
+        <a href="#" class="brand-logo center">Tabs VS Spaces</a>
+    </div>
+</nav>
+<div class="section">
+    <div class="center">
+        <h4>
+            <c:choose>
+                <c:when test="${tabCount == spaceCount}">
+                    TABS and SPACES are evenly matched!
```
Comment (kolea2) [inline cloud-sql/sqlserver/servlet/src/main/webapp/index.jsp:37]: it might be nice to explain somewhere (maybe the README?) what this example app is doing

**Review by kolea2 — CHANGES_REQUESTED** [review]

File: cloud-sql/sqlserver/servlet/pom.xml (hunk 2 above)
Comment (shubha-rajan) [reply cloud-sql/sqlserver/servlet/pom.xml]: oops! forgot to set this back to the placeholder value. fixed this and the other occurrence!

File: cloud-sql/sqlserver/servlet/src/main/java/com/example/cloudsql/IndexServlet.java (hunk 4 above)
Comment (shubha-rajan) [reply cloud-sql/sqlserver/servlet/src/main/java/com/example/cloudsql/IndexServlet.java:132]: Should be warning, that's what it is in the other samples. The next option above "WARNING" is "SEVERE" and idk if that's right here.

**Comment by shubha-rajan** [comment]
Let's hold off on merging this. I'm noticing an issue with the App Engine local server after updating the maven-appengine-plugin version. Deploying works fine, but the dev server fails to start. I will investigate.

**Comment by shubha-rajan** [comment]
> Let's hold off on merging this. I'm noticing an issue with the App Engine local server after updating the maven-appengine-plugin version. Deploying works fine, but the dev server fails to start. I will investigate.

nvm it was silly. I was using the wrong Java version in my shell. Switching to Java 8 fixed it.

**Review by kurtisvg — APPROVED** [review]

**Comment by shubha-rajan** [comment]
@kolea2 good to merge?

**Review by kolea2 — APPROVED** [review]
lgtm!

**Pull request merged**